app/assets/stylesheets/logins.css

```diff
@@ -0,0 +1,4 @@
+/*
```

Comment:
Ideally, we'll only need one stylesheet for the whole app—the theme/styles should be the same across pages, and the only differences will be sections that we can determine with classes or ids.